modules/telemetry-otel/src/main/java/org/opensearch/telemetry/OTelTelemetryModulePlugin.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.telemetry;
+
+import org.opensearch.common.settings.Setting;
+import org.opensearch.common.settings.Settings;
+import org.opensearch.common.unit.TimeValue;
+import org.opensearch.common.util.FeatureFlags;
+import org.opensearch.plugins.Plugin;
+import org.opensearch.plugins.TelemetryPlugin;
+import org.opensearch.telemetry.metrics.MetricsTelemetry;
+import org.opensearch.telemetry.tracing.OtelTelemetryImpl;
+import org.opensearch.telemetry.tracing.OtelTracingTelemetry;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.opensearch.common.util.FeatureFlags.TELEMETRY;
+
+/**
+ * Telemetry plugin based on Otel
+ */
+public class OTelTelemetryModulePlugin extends Plugin implements TelemetryPlugin {
+
+    static final String OTEL_TRACER_NAME = "otel";
+
+    static final Setting<Integer> TRACER_EXPORTER_BATCH_SIZE_SETTING = Setting.intSetting(
```

Comment:
Could you please make these settings module specific (since this is `otel` module), fe `telemetry.otel.tracer.exporter.batch_size`